.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please remove these settings. Because RAN is using prettier+eslint integration.

Reply:
Relative to this, I already removed it, but can you consider use .prettierrc file for prettier settings? Because I think its the best way we can share the prettier setting, if not, I have to configure it on VSCode each time for specific project. This is quoted from the eslint-plugin-prettier README.md:
 "Note: While it is possible to pass options to Prettier via your ESLint configuration file, it is not recommended because editor extensions such as prettier-atom and prettier-vscode will read .prettierrc, but won't read settings from ESLint, which can lead to an inconsistent experience."
If you want to use it, I can work on that issue too 😄